Framework/PythonInterface/mantid/api/src/Exports/AnalysisDataService.cpp

```diff
@@ -40,7 +40,7 @@ AnalysisDataServiceImpl &instance() {
   std::call_once(INIT_FLAG, []() {
     PyRun_SimpleString("import atexit\n"
                        "from mantid.api import AnalysisDataService\n"
-                       "atexit.register(lambda: AnalysisDataService.clear())");
+                       "atexit.register(lambda: AnalysisDataService.clear(), True)");
```

Comment:
Please add a comment about what the "True" is doing. It is very clear in context of this PR, but I don't think it will make sense when seen in the wild.